SU-SOE/stanford_soe_helper
# 7.x 2.x soe 2923

# READY FOR REVIEW

# Summary
This adds the Injector file 41 into SCSS.

# Needed By (Date)
- Sprint end

# Criticality
- Collections Injector file 41 into SCSS


# Steps to Test
- Switch to this branch  -  7.x-2.x-SOE-2923 (https://github.com/SU-SOE/stanford_soe_helper/tree/7.x-2.x-SOE-2923)
- edit the injector: `admin/config/development/css-injector/edit/41`
- `drush cc css-js`
- for the collection styles go to: `magazine/future-everything` and make sure the changes look good.
- for all the rest of the styles go to: `magazine/article/jeremy-bailenson-taking-grand-tour-latest-virtual-reality` and make sure the changes look good.

# Injector code and screen shots
`.node-type-stanford-collection #mc_embed_signup_scroll {
    padding: 0;
}

.node-type-stanford-collection .mailchimp-magazine-block #mc_embed_signup_scroll {
    padding: 65px 0;
}`

![screen shot 2018-06-21 at 4 54 37 pm](https://user-images.githubusercontent.com/571421/41747550-f6763b0c-7573-11e8-97a3-8a2dd6907226.png)

`.page-magazine .mailchimp-magazine-block #mc_embed_signup_scroll,
.page-magazine-all .mailchimp-magazine-block #mc_embed_signup_scroll,
.page-taxonomy-term .mailchimp-magazine-block #mc_embed_signup_scroll {
  margin-bottom: 0;
}`

![screen shot 2018-06-21 at 5 00 42 pm](https://user-images.githubusercontent.com/571421/41747771-c8932f78-7574-11e8-9b78-52ecc3fae5da.png)

`.page-magazine form#mc-embedded-subscribe-form,
.page-magazine-all form#mc-embedded-subscribe-form,
.page-taxonomy-term form#mc-embedded-subscribe-form,
.node-type-stanford-collection form#mc-embedded-subscribe-form {
  margin-bottom: 0;
}`

![screen shot 2018-06-21 at 5 01 10 pm](https://user-images.githubusercontent.com/571421/41747775-cfbaba32-7574-11e8-8e0b-6b40f3001d0e.png)

`.page-magazine .mailchimp-magazine-block #mc_embed_signup_scroll,
.page-magazine-all .mailchimp-magazine-block #mc_embed_signup_scroll,
.page-taxonomy-term .mailchimp-magazine-block #mc_embed_signup_scroll,
.node-type-stanford-collection .mailchimp-magazine-block #mc_embed_signup_scroll {
  box-shadow: none;
  -webkit-box-shadow: none;
}`

![screen shot 2018-06-21 at 5 00 42 pm](https://user-images.githubusercontent.com/571421/41747771-c8932f78-7574-11e8-9b78-52ecc3fae5da.png)

`@media (max-width: 767px) {
  .page-magazine .mailchimp-magazine-block #mc_embed_signup_scroll,
  .page-magazine-all .mailchimp-magazine-block #mc_embed_signup_scroll,
  .page-taxonomy-term .mailchimp-magazine-block #mc_embed_signup_scroll,
  .node-type-stanford-collection .mailchimp-magazine-block #mc_embed_signup_scroll {
    padding: 40px 0;
  }
}`

![screen shot 2018-06-21 at 5 04 52 pm](https://user-images.githubusercontent.com/571421/41747954-6ae45df6-7575-11e8-83b8-1193a1cf212e.png)

`.page-magazine #content-body-bottom,
.page-magazine-all #content-body-bottom,
.page-taxonomy-term #content-body-bottom,
.node-type-stanford-collection #content-body-bottom {
  margin-top: 5%;
}`

![screen shot 2018-06-21 at 5 07 48 pm](https://user-images.githubusercontent.com/571421/41748018-a7a9e7c4-7575-11e8-92b5-8f7d6e3c8e62.png)

`/* block width */ 
.node-type-stanford-magazine-article #block-bean-stanford-soe-mag-news-signup .mailchimp-magazine-block #mc_embed_signup_scroll { 
  width: 50%;
  margin: 0 auto;
}`

![screen shot 2018-06-21 at 5 09 20 pm](https://user-images.githubusercontent.com/571421/41748071-e6d04204-7575-11e8-9a4d-7662c165d70d.png)

`/* h2 font size */ 
.node-type-stanford-magazine-article #block-bean-stanford-soe-mag-news-signup .mailchimp-magazine-block #mc_embed_signup_scroll h2 { 
  font-size: 1.3em;
  margin-bottom: 20px;
}`

![screen shot 2018-06-21 at 5 10 14 pm](https://user-images.githubusercontent.com/571421/41748099-02f48ecc-7576-11e8-9a9a-e2b384e754a6.png)

`/* input width */ 
.node-type-stanford-magazine-article #block-bean-stanford-soe-mag-news-signup .mailchimp-magazine-block #mc_embed_signup_scroll .mc-field-group input {
  width: 300px;
}`

![screen shot 2018-06-21 at 5 11 12 pm](https://user-images.githubusercontent.com/571421/41748129-22e106de-7576-11e8-81f1-bb32cde92a26.png)

`/* block width */ 
@media (max-width: 979px) {
  .node-type-stanford-magazine-article #block-bean-stanford-soe-mag-news-signup .mailchimp-magazine-block #mc_embed_signup_scroll {
    width: 70%;
    }
}`

![screen shot 2018-06-21 at 5 12 12 pm](https://user-images.githubusercontent.com/571421/41748158-44422e3e-7576-11e8-9488-98ab5e4ffdc0.png)

`/* input width */ 
@media (max-width: 580px) {
  .node-type-stanford-magazine-article #block-bean-stanford-soe-mag-news-signup .mailchimp-magazine-block #mc_embed_signup_scroll .mc-field-group input {
    width: 200px;
  }
}`

![screen shot 2018-06-21 at 5 13 45 pm](https://user-images.githubusercontent.com/571421/41748211-7b5eef1a-7576-11e8-9673-79d6d668f600.png)

`@media (max-width: 480px) {
/* block width & padding */ 
  .node-type-stanford-magazine-article #block-bean-stanford-soe-mag-news-signup .mailchimp-magazine-block #mc_embed_signup_scroll {
    width: 80%;
    padding: 25px;
  }
/* h2 font-size */ 
  .node-type-stanford-magazine-article #block-bean-stanford-soe-mag-news-signup .mailchimp-magazine-block #mc_embed_signup_scroll h2 {
    font-size: 1.1em;
  }
/* input width */ 
  .node-type-stanford-magazine-article #block-bean-stanford-soe-mag-news-signup .mailchimp-magazine-block #mc_embed_signup_scroll .mc-field-group input {
    width: 150px;
  }  
}`

NO SCREEN SHOT - couldn't get my browser that small, but it's there :)

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
https://stanfordits.atlassian.net/browse/SOE-2923

## Related PRs

## More Information

## Folks to notify
@cjwest and @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
